css/properties/-webkit-locale.json

```
@@ -5,7 +5,7 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

See: https://github.com/caugner/css-properties/commit/89455fe987fbd89d0c77bc9fd38454db83724c6e

```suggestion
              "version_added": "11"
```